generators/assets/webpack/templates/webpack.config.js.tmpl

```diff
@@ -44,48 +104,6 @@ module.exports = {
         screw_ie8: true
       },
       comments: false
-    }),
-    new ManifestPlugin({
```

Comment:
manifest is needed for application.[hash].js to work

Reply:
It’s still there. Look at the whole file.

Reply:
ohh ya